app/scripts/modules/netflix/fastProperties/domain/canary.domain.ts

```diff
@@ -0,0 +1,87 @@
+
+import {PropertyCommand} from './propertyCommand.model';
+import {IUser} from 'core/authentication/authentication.service';
+import {PropertyCommandType} from './propertyCommandType.enum';
+import {ICanaryDeployment} from './canaryDeployment.interface';
+import {AcaTaskStageConfigDetails} from './acaTaskStageConfigDetails.model';
+
+export class Canary {
+  public application: string = 'spinnakerfp';
+  public owner: string;
+  public canaryConfig: CanaryConfig;
+  public watchers: string[] = [];
+  public canaryDeployments: CanaryDeployments[];
+
+  constructor(private user: IUser, private command: PropertyCommand, private configDetails: AcaTaskStageConfigDetails ) {
```

Comment:
since this class has no private methods, we can remove the `private` access modifier on the ctor args as there is no internal reference to the properties.